Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,10 +1,30 @@
 import React from 'react';
 import './Header.css';
 
+let day = new Date().getDate();
```

Comment:
Great, job coming to a solution for displaying the time here. Lots of ways to get this done both w/vanillaJS and a library like moment.js